service/frontend/handler.go

```diff
@@ -521,6 +523,61 @@ func (wh *WorkflowHandler) RespondActivityTaskCompleted(
 	return nil
 }
 
+// RespondActivityTaskCompletedByID - response to an activity task
+func (wh *WorkflowHandler) RespondActivityTaskCompletedByID(
+	ctx context.Context,
+	completeRequest *gen.RespondActivityTaskCompletedByIDRequest) error {
+
+	scope := metrics.FrontendRespondActivityTaskCompletedScope
```

Comment:
Probably you want to create a separate metric scope for this API.

Reply:
Make sense. Updated